tests/Application/src/Entity/BookTranslationInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Sylius\Resource\Model\ResourceInterface;
+
+interface BookTranslationInterface extends ResourceInterface
```

Comment:
```suggestion
interface BookTranslationInterface extends ResourceInterface, TranslationInterface
```

Reply:
Yes, I've noticed that too. The AbstractTranslation already implements that TranslationInterface, so normally it will not change anything but I could change that to reflect better a real use case.